<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerShapePopulationViewerModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerShapePopulationViewerModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="ModuleWidgetLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ModelLabel">
       <property name="text">
        <string>Select Model:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="ModelNodeComboBox">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLModelNode</string>
        </stringlist>
       </property>
       <property name="showChildNodeTypes">
        <bool>false</bool>
       </property>
       <property name="addEnabled">
        <bool>false</bool>
       </property>
       <property name="removeEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ModelLoadPushButton">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="ImportCTKCollapsibleButton">
     <property name="text">
      <string>Import</string>
     </property>
     <layout class="QVBoxLayout" name="ImportLayout">
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="ExportCTKCollapsibleButton">
     <property name="text">
      <string>Export</string>
     </property>
     <layout class="QHBoxLayout" name="ExportLayout">
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="SettingsCTKCollapsibleButton">
     <property name="text">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="SettingsLayout">
      <item>
       <widget class="QCheckBox" name="ModelAutomaticLoadingCheckBox">
        <property name="text">
         <string>Automatically load models</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerShapePopulationViewerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ModelNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
